### DISCUSSION TITLE
레벨1 글쓰기 "한 문장 쓰기" | 2023.03.02

### DISCUSSION THREAD

**Discussion by Wonmi91** [woowacourse/retrospective#31] [General]
## 레벨 1 글쓰기 주제 : 우아한테크코스 한 달 생활기
- 레벨 1 글쓰기의 첫 문장을 지금 생각나는대로 써보자!
- 최종 글쓰기에 꼭 이 문장을 반영할 필요는 없다 : )

## 이전 기수 크루들은 첫 문장을 어떻게 썼을까?
- 지금까지 내 인생은 ‘뒤로 가기’였다.
- 사람은 스스로 엮은 의미의 그물 속에서 살아가는 물고기라고 한다. 우아한테크코스에 오기 전과 오고 나서 겪었던 일들을 내 삶을 지탱하는 명언들과 엮은 그물을 만들어 나의 우아한테크코스 한 달 생활기를 적어보려고 한다.
- 작년 8월, 4년을 다닌 회사에 사표를 내고 꿈꾸던 일을 해보겠다며 프로그래밍 공부를 시작했다. 
- "나? 요즘 안 바빠… 그냥 조금 정신이 없네."
- 우테코에는 내 이름이 없다.
- “그래서 뭘 배웠어?” 얼마 전 우아한 테크코스에 대해 지인에게 이야기하던 도중 받았던 질문이다.
- 당신은 잘 지내시나요? 코로나 때문에 시작된 재택 교육이 계속 길어지네요.

**Comment by s9hn** [comment]
추운 바람과 따듯한 봄기운이 마주하는 겨울이었다.

**Comment by junpakPark** [comment]
11개월 간 매주 10,080원을 주면 어떻게 쓰실 건가요?

**Comment by shin-mallang** [comment]
부딪히고, 좌절하고, 발전해라.

**Comment by shb03323** [comment]
오늘의 나는 나에게 떳떳했나요?

**Comment by hyeonjerry** [comment]
세상을 등지고 새로운 세상을 품었다.

**Comment by hyena0608** [comment]
이래야 우테코지

**Comment by tmdgh1592** [comment]
오늘 귀찮은 건 내일도 귀찮다.

**Comment by This2sho** [comment]
요즘 매일 가면을 쓴다.

**Comment by TaeyeonRoyce** [comment]
우테코맛 칵테일

**Comment by cpot5620** [comment]
월요일이 좋아

**Comment by GC-Park** [comment]
어색함으로 시작한 우테코, 익숙함으로 물든 한 달이었다.

**Comment by rawfishthelgh** [comment]
- 매일 오늘 저녁 밥을 누구랑 먹을지가 고민이었다.

**Comment by kong-hana01** [comment]
"새는 알에서 나오기 위해 투쟁한다."

**Comment by ukkodeveloper** [comment]
나도 튀겨질 수 있을까?

**Comment by carsago** [comment]
저번에 마신 생맥이 떠오른다.

**Comment by krrong** [comment]
우리가 현재의 모습이 아닌 다른 모습일 수도 있다는 느낌이야말로 불안과 울화의 원천이다.

**Comment by Choi-JJunho** [comment]
"안녕하세요? 저는 주노라고합니다 혹시 닉네임이 어떻게되세요?"

**Comment by kpeel5839** [comment]
날 죽이지 못하는 고통은 나를 성장시킨다.

**Comment by BGuga** [comment]
나는 남보다 날 모른다.

**Comment by the9kim** [comment]
형제는 우아하지 않다.

**Comment by jyeost** [comment]
내 인생 어쩌면 순풍을 탔는지도 ?

**Comment by turtle601** [comment]
우테코에서 살아남기!!

**Comment by yoondgu** [comment]
2월 7일부터 나에게 새로운 이름이 생겼다.

**Comment by pilyang** [comment]
지금까지 혼자 걷던 길을 같이 걸어갈 동료들이 생겼다.

**Comment by carsago** [comment]
저번에 마셨던 생맥이 생각난다.

**Comment by SeongHoonC** [comment]
오늘도 커피를 마신다.

**Comment by briandr97** [comment]
마음만 먹으면 못할게 없다! 글쓰기도 그렇다!

**Comment by Songusika** [comment]
지금이 당신이 그토록 기대하던 우아한테크코스 생활인가요?

**Comment by rhthrhrl0** [comment]
주변 환경의 힘이 얼마나 큰지 새삼 깨달았다

**Comment by 2yunseong** [comment]
자신이 없다 ... 질 자신이

**Comment by kyw0716** [comment]
요즘 내 꿈은 명예로운 고졸이다.

**Comment by chsua** [comment]
불확실성에서 부유하기

**Comment by iamjooon2** [comment]
1년 뒤에 봐

**Comment by kevstevie** [comment]
연극은 몰래카메라인줄 알았는데

**Comment by CFalws** [comment]
긴 경부선을 빠져나오자, 잠실이었다.

**Comment by chaewon121** [comment]
티클 같은 하루가 모여 태산이 된다.

**Comment by 2chang5** [comment]
사람은 죽어서 이름을 남기고 멧돼지는 죽어서 가죽을 남긴다 (멧돼지의 삶의 교훈)

**Comment by ingpyo** [comment]
"잠,,, 그게 뭔데"

**Comment by otter66** [comment]
매일매일 행복한 하루들

**Comment by inyeong-kang** [comment]
이게 되네?

**Comment by hectick** [comment]
날 ooh ahh ooh ahh 하게 만들어줘 ~

**Comment by seokhwan-an** [comment]
수동적인 사람이 아닌 능동적인 사람이 되어야 했다

**Comment by feb-dain** [comment]
바쁨과 즐거움이 공존하는 한 달

**Comment by wugawuga** [comment]
01036551086 우가 입니다. 문자 주세요

**Comment by SproutMJ** [comment]
성장 곡선의 기울기가 올라간다.

**Comment by sh111-coder** [comment]
감정은 흔들리지만 생활은 흔들리지 않는다

**Comment by gitchannn** [comment]
우테코를 시작하고 월요일이 좋은걸 처음으로 알았다

**Comment by inseonyun** [comment]
우당탕탕 I의 우테코 생존기

**Comment by yeopto** [comment]
시궁창같은 삶에서 "우아"한 삶을 살아보려한다.

**Comment by hanueleee** [comment]
불이야 불이야

**Comment by woo-chang** [comment]
내 계획은 10개월 짜리니까 기대해라

**Comment by tkdrb12** [comment]
지금까지 내 삶은 우아하지 않았다...

**Comment by echo724** [comment]
echo "우아한테크코스에 울리는 메아리"

**Comment by Kim0914** [comment]
예상치 못한 일의 연속

**Comment by dooboocookie** [comment]
행복한데 불안하다.

**Comment by pingu244** [comment]
이제서야 시작된 본격적인 우테코 생활

**Comment by 0chil** [comment]
너는 다 계획이 있구나?

**Comment by kiarakim** [comment]
몇년을 꿈 꾼 미라클 모닝을 시작했다

**Comment by DYGames** [comment]
팝니다 : 노트북, 5년 사용함

**Comment by jundonghyuk** [comment]
"페어 폼 미쳤다"

**Comment by ashleysyheo** [comment]
매일 잠이 부족하다

**Comment by hafnium1923** [comment]
연습삼아 도전했던 시험이 덜컥 붙었을 때의 기분을 아시나요?

**Comment by HubCreator** [comment]
드디어 한발짝이다.

**Comment by Cyma-s** [comment]
나는 과거의 나를 부수기 위해 도전한다

**Comment by tjdtls690** [comment]
10개월간 매일 여행을하면 어떤 기분일까?

**Comment by jeomxon** [comment]
더욱 커지는 불신은 많은 생각을 갖게 한다.

**Comment by hyemdooly** [comment]
그 누구도 내가 아침형 인간이 될 수 있을줄 몰랐다

**Comment by odo27** [comment]
배가 고프다

**Comment by java-saeng** [comment]
나 행복해 진짜야

**Comment by hozzijeong** [comment]
"connecting the dots", 점들을 잇는 방법에 대해 배우다.

**Comment by chws0508** [comment]
드디어 시작이다

**Comment by jeonjeunghoon** [comment]
위태로운 삶이었다.

**Comment by WaiNaat** [comment]
그는 겁쟁이였고, 겁쟁이고, 겁쟁이일 것이다.

**Comment by donghae-kim** [comment]
내일의 나를 만드는 우테코

**Comment by nlom0218** [comment]
30년간 소심한 먼지로 살아갔지만 이제는 아니다.

**Comment by re4rk** [comment]
월세 낸게 아까워서 열심히한다.

**Comment by hyunji1203** [comment]
"하면해"하다가 우테코까지 들어왔다

**Comment by yeonkkk** [comment]
어색하고 즐거운 우테코 생활

**Comment by Jaeyoung22** [comment]
나는 우테코를 믿지 않는다.

**Comment by MoonJeWoong** [comment]
내 손에는 지금 몇 개의 '코인'이 있을까?

**Comment by Gilpop8663** [comment]
이번 한 달간 지내본 결과 선릉에 올때마다 긍정적인 생각이 들었다

**Comment by woowajames** [comment]
"아이스 아메리카노 한 잔이요."

**Comment by ki960213** [comment]
우테코 쨔응

**Comment by xodms0309** [comment]
10개월의 대장정 시작

**Comment by yujamint** [comment]
공통된 한 가지에 몰입하는 사람이 100명 이상 모였을 때 생기는 힘은 크다.

**Comment by jw-r** [comment]
순간이동 배우고 싶다

**Comment by whk06061** [comment]
눈 떠보니 벌써 한달이 훌쩍

**Comment by thdwoqor** [comment]
운이 좋았다.

**Comment by wzrabbit** [comment]
아무것도 모르는, 어리둥절한 견습 마법사가 우테코에 합류했어요!

**Comment by cookienc** [comment]
바쁘다, 힘들다 그렇지만 재밌다.

**Comment by somsom13** [comment]
대학교 5학년을 다니는 기분이다.

**Comment by Ohjintaek** [comment]
우테코는 마치 '산업용 다이아몬드'를 만드는 것과 비슷하다고 느꼈다

**Comment by waterricecake** [comment]
물빠진떡은 여전히 싫다.

**Comment by Dahyeeee** [comment]
"재밌겠다"는 섣부른 판단이었다.

**Comment by 70825** [comment]
하루가 빠르게 지나가고 있다

**Comment by woosung1223** [comment]
포비를 실물로 영접하다니

**Comment by jjongwa** [comment]
나는 박치기 공룡이다.

**Comment by kdkdhoho** [comment]
맘놓고 놀길 잘했다

**Reply by 2chang5** [reply]
파키케팔로사우르스

**Comment by nangkyeonglim** [comment]
잠에 들고 1분 후에 눈을 뜨니 아침이다.

**Comment by jiwonh423** [comment]
즐거운 우테코~

**Comment by hgo641** [comment]
하루에 토론만 6시간;

**Comment by JJ503** [comment]
바쁘다 바빠 현대사회

**Comment by sh981013s** [comment]
익숙함에 과거의 간절함을 잊지말자.

**Comment by guridaek** [comment]
행운은 찾아오는 게 아니라 찾으러 가는 것이다.

**Comment by regularPark** [comment]
나 혹시 전산 오류 전형?

**Comment by sosow0212** [comment]
지금 머리는 러닝머신 중이다.

**Comment by ippnsj** [comment]
우테코를 시작하면서 내 하루하루가 가득차게 되었다.

**Comment by geuntaek1013** [comment]
달면 삼키고, 써도 삼켜야한다.

**Comment by splitCoding** [comment]
지금의 나를 아껴야 할까 미래의 나를 아껴야 할까

**Comment by rhkrwngud445** [comment]
부산에서 출발한 긴 여정

**Comment by aiaiaiai1** [comment]
나라는 사람 찾아가기

**Comment by drunkenhw** [comment]
나는 성격이 더럽다

**Comment by n0eyes** [comment]
컴포즈 커피 쿠폰이 벌써 4개다

**Comment by Cl8D** [comment]
우테코에서 내향인은 살아남을 수 없다

**Comment by Combi153** [comment]
어쩌다가 개발자 같은 걸 하게 됐을까

**Comment by yenawee** [comment]
올해는 흘러가는대로 살아가면 안될거같다

**Comment by woo-jk** [comment]
간절했던 우테코, 여전히 간절한가?

**Comment by shackstack** [comment]
눈떠보니 선릉역 10번 출구 앞이야...

**Comment by Songusika** [comment]
내 인생 리팩토링

**Comment by amaran-th** [comment]
우테코는 스펙 이력이 아닌 기회일 뿐 그 기회를 잡을지는 오로지 나의 몫이다.

**Comment by apptie** [comment]
언제쯤 적응할 수 있을까

**Comment by mcodnjs** [comment]
매일 새롭다

**Comment by ksone02** [comment]
언제쯤 나의 배터리는 100%가 될까

**Comment by Leejin-Yang** [comment]
점메추 받아요

**Comment by suyoungj** [comment]
애물단지 인텔 맥북을 10개월만 더 쓰기로 했다.

**Comment by no1msh** [comment]
늘 하던 것들만 하면 늘 얻던 것들을 얻는다.

**Comment by jaehee329** [comment]
남들 보니 내가 이러고 있으면 안 될 것 같은데

**Comment by kangyeongmin** [comment]
어쩌다 여기에 있는거지?

**Comment by kong-hana01** [comment]
오빠 맥북 있어?

**Comment by gyeongza** [comment]
어?

**Comment by zillionme** [comment]
지도 하나 들고 자유 여행하는 느낌

**Comment by solo5star** [comment]
나답게!

**Comment by eunbii0213** [comment]
우테코에서 최약체로 살아남기🐰 가보자고~!

**Comment by D0Dam** [comment]
아직 도담 모르시는 분 있나요? 😆

**Comment by LJW25** [comment]
나는 비열이 큰 사람인줄 알았는데, 열정을 가진 사람들의 에너지는 그 이상이었다.

**Comment by greeng00se** [comment]
허브티🌿 한 잔 하실래요?

**Comment by dladncks1217** [comment]
"기회는 언제든지 올 수 있다." 준비된 사람이 되기 위한 우테코 1달 생활기

**Comment by RightHennessy** [comment]
아 그냥 배민에서 하는 학원 같은거야 ~

**Comment by be-student** [comment]
오늘 점심 같이 먹을 분 구해요

**Comment by ippnsj** [comment]
매일 아침 신분당선이 제시간에 오길 기도한다... 😭

**Comment by seokjin8678** [comment]
잘못된 선택은 없었다.

**Comment by hum02** [comment]
시작하면 개발공부만 할 줄 알았다.

**Comment by yogjin** [comment]
아 배고파

**Comment by jeongwusi** [comment]
포기하지 않으면 프로그래밍은 누구나 할 수 있어요.  - 포비 -

**Comment by bassyu** [comment]
지금 쓰는 이 글도 우테코의 교육이다

**Comment by Go-Jaecheol** [comment]
나는 초짜니까

**Comment by semnil5202** [comment]
내일 아침이 기다려진다

**Comment by jariita** [comment]
실패를 두려워하지 마라

**Comment by parkmuhyeun** [comment]
"2023년 2월 7일 00시 00분 67.75kg의 초우량아가 태어났다"

**Comment by poi1649** [comment]
이번 주 일일 평균 스크린 타임은 1시간 40분입니다. 지난 주 대비 71% 감소했습니다.
